schemas/IDLs/org.gel.models.report.avro/4.0.0/CommonRequest.avdl

```diff
@@ -39,7 +42,10 @@ Types:
         BigWig,
         MD5Sum,
         ROH,
-        OTHER
+        OTHER,
+        PARTITION,
+        VARIANT_FREQUENCIES,
+        COVERAGE
```

Comment:
@priesgo are you happy with the uppercase ?

Reply:
I am